eclipse-bluechi/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# /********************************************************************************
+# * Copyright (c) 2024 Leonardo Rossetti <lrossett@redhat.com>
+# *
+# * This program and the accompanying materials are made available under the 2
+# * terms of the Apache License 2.0 which is available at
+# * https://www.apache.org/licenses/LICENSE-2.0.txt.
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# *
+# * Contributors: 1
+# *   Leonardo Rossetti - file creation
+# ********************************************************************************/
+FROM quay.io/centos-sig-automotive/autosd:latest
+
+ARG GRPCURL_URL=https://github.com/fullstorydev/grpcurl/releases/download/v1.8.7/grpcurl_1.8.7_linux_x86_64.tar.gz
```

Comment:
Should we consider arm64 as well? And likewise does the base image support arm? One way to accomplish this is using TARGETARCH. Example can be found here: https://github.com/eclipse-sdv-blueprints/software-orchestration/blob/5e5e06308629e7737e3c7a9651db062309b5af36/scenarios/smart_trailer/Dockerfile.sample_workloads#L41C1-L53C71

Reply:
that image contains both archs and should pull the correct arch based on the host architecture.

Let me add a var to pull that binaries for both archs